src/Misc/layoutroot/run.sh

```diff
@@ -9,16 +9,28 @@ while [ -h "$SOURCE" ]; do # resolve $SOURCE until the file is no longer a symli
     [[ $SOURCE != /* ]] && SOURCE="$DIR/$SOURCE" # if $SOURCE was a relative symlink, we need to resolve it relative to the path where the symlink file was located
 done
 DIR="$( cd -P "$( dirname "$SOURCE" )" && pwd )"
+
+# Set job control
+set -m
+
+trap 'kill -INT -$PID' INT TERM
```

Comment:
I am wondering whether we should wrap these new codes under some env to avoid any widespread impact.

Reply:
ex: `RUNNER_MANUALLY_TRAP_SIG`

Reply:
That would make more sense, to be honest. This is a bit risky. I have tested the self update, and some cases manually, but to make sure we are not breaking anything. Maybe it would be better to expose an env variable that will run like this, and if not set, run everything like before.
Should I do that?

Reply:
I think we should do it, feature flag this behind some ENV.